### PR TITLE
Add UBERON:8410011 inguinal region skin

### PR DESCRIPTION
**Pull request by paolaroncaglia**
Addresses #1688 (not 1668 as the erroneous branch name would suggest)

### CODE REVIEW

File: uberon_edit.obo [hunk 1]
```diff
@@ -195227,6 +195227,25 @@ relationship: dc-creator http://orcid.org/0000-0003-2034-601X
 relationship: part_of UBERON:0004647 ! liver lobule
 property_value: http://purl.org/dc/elements/1.1/date 2020-10-05T15:52:22Z xsd:dateTime
 
+[Term]
+id: UBERON:8410011
+name: inguinal region skin
+def: "The skin covering the inguinal region between the abdomen and the thigh on either side of the pubic bone." []
```
Comment (dosumis) [inline uberon_edit.obo:195233]: Missing xref.  You could use the versions wikipedia link currently in xref

File: uberon_edit.obo [hunk 2]
```diff
@@ -195227,6 +195227,25 @@ relationship: dc-creator http://orcid.org/0000-0003-2034-601X
...
+synonym: "groin skin" EXACT []
+synonym: "inguinal skin" EXACT []
+synonym: "skin of groin" EXACT []
+synonym: "skin of inguinal region" EXACT []
+synonym: "zone of skin of groin" EXACT []
+synonym: "zone of skin of inguinal region" EXACT []
+xref: http://orcid.org/0000-0001-7505-5418
+xref: https://en.wikipedia.org/w/index.php?title=Groin&oldid=998393002
```
Comment (dosumis) [inline uberon_edit.obo:195241]: Should be curie.  Check GO xref abbs for prefix.

**Review by dosumis — CHANGES_REQUESTED** [review]
see rqd changes in comments.  Term itself looks good.

File: uberon_edit.obo (hunk 2 above)
Comment (paolaroncaglia) [reply uberon_edit.obo:195241]: AI for @dosumis : figure out curie format for Wikipedia revision history (so it's picked up correctly by OLS) and report.
AI for @paolaroncaglia : then fix all Wikipedia links in all my PRs/terms.

Comment (dosumis) [reply uberon_edit.obo:195241]: Please use WikipediaVersioned:Groin&oldid=998393002

**Comment by paolaroncaglia** [comment]
@dosumis 
All done, thanks.

**Comment by dosumis** [comment]
Build now fixed.  One last request (apologies for missing this before) - could you delete asserted classifications when the reasoner can infer the.  Checked this one and it is inferred.  This => dynamic classification that doesn't go stale.

**Comment by paolaroncaglia** [comment]
@dosumis 

> Build now fixed. One last request (apologies for missing this before) - could you delete asserted classifications when the reasoner can infer the. Checked this one and it is inferred. This => dynamic classification that doesn't go stale.

Now done, sorry I missed that one yesterday. I haven't tried to resolve the conflicts yet.

**Pull request closed without merging**